new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/MySQLDBUtils.java

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.utils;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import io.kubernetes.client.custom.IntOrString;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1Container;
+import io.kubernetes.client.openapi.models.V1ContainerPort;
+import io.kubernetes.client.openapi.models.V1EnvVar;
+import io.kubernetes.client.openapi.models.V1EnvVarSource;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodSpec;
+import io.kubernetes.client.openapi.models.V1Secret;
+import io.kubernetes.client.openapi.models.V1SecretKeySelector;
+import io.kubernetes.client.openapi.models.V1Service;
+import io.kubernetes.client.openapi.models.V1ServicePort;
+import io.kubernetes.client.openapi.models.V1ServiceSpec;
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.actions.impl.Namespace;
+import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Utility class to create MySQL database.
+ */
+public class MySQLDBUtils {
+
+  /**
+   * Create and start a MySQL database pod.
+   *
+   * @param name      name of the db pod
+   * @param user      username for the database
+   * @param password  password for the database
+   * @param nodePort  node port of db service
+   * @param namespace name of the namespace in which to create MySQL database
+   */
+  public static void createMySQLDB(String name, String user, String password, int nodePort, String namespace) {
+
+    String uniqueName = Namespace.uniqueName();
+    String secretName = name.concat("-secret-").concat(uniqueName);
+    String serviceName = name.concat("-external-").concat(uniqueName);
+
+    createSecret(secretName, user, password, namespace);
+    createService(serviceName, name, namespace, nodePort);
+    startMySQLDB(name, secretName, namespace);
+
+  }
+
+  //create the database pod
+  private static void startMySQLDB(String name, String secretName, String namespace) {
+    Map<String, String> labels = new HashMap<>();
+    labels.put("app", name);
+    V1Pod mysqlPod = new V1Pod()
+        .metadata(new V1ObjectMeta()
+            .name(name)
+            .namespace(namespace)
+            .labels(labels))
+        .spec(new V1PodSpec()
+            .terminationGracePeriodSeconds(5L)
+            .containers(Arrays.asList(new V1Container()
+                .image("mysql:5.6")
```

Comment:
should we have have image as param in case we use different versions?

Reply:
Good suggestion. Added a new param for taking the MySQL image version.